src/utils/ReactDOMMini.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import ReactReconciler from 'react-reconciler';
+import * as _ from 'lodash';
+import ReactDOMServer from 'react-dom/server';
+
+const toReactElement = (e) => {
+  if (e.children.length === 0) {
+    return React.createElement(e.type, e.props);
+  }
+  return React.createElement(
+    e.type,
+    e.props,
+    e.children.map((c) => (typeof c === 'string' ? c : toReactElement(c))),
+  );
+};
+
+const matchHtmlRegExp = /["'&<>]/;
+
+function escapeHtml(string) {
+  const str = '' + string;
+  const match = matchHtmlRegExp.exec(str);
+
+  if (!match) {
+    return str;
+  }
+
+  let escape;
+  let html = '';
+  let index;
+  let lastIndex = 0;
+
+  for (index = match.index; index < str.length; index++) {
+    switch (str.charCodeAt(index)) {
+      case 34: // "
+        escape = '&quot;';
+        break;
+      case 36: // $
+        escape = '&#36;';
+        break;
+      case 38: // &
+        escape = '&amp;';
+        break;
+      case 39: // '
+        escape = '&#x27;'; // modified from escape-html; used to be '&#39'
+        break;
+      case 60: // <
+        escape = '&lt;';
+        break;
+      case 62: // >
+        escape = '&gt;';
+        break;
+      default:
+        continue;
+    }
+
+    if (lastIndex !== index) {
+      html += str.substring(lastIndex, index);
+    }
+
+    lastIndex = index + 1;
+    html += escape;
+  }
+
+  return lastIndex !== index ? html + str.substring(lastIndex, index) : html;
+}
+
+function escapeTextForBrowser(text) {
+  if (typeof text === 'boolean' || typeof text === 'number') {
+    return '' + text;
+  }
+  return escapeHtml(text.trim());
+}
+
+const reconciler = ReactReconciler({
+  supportsMutation: true,
```

Comment:
Do we really need to support mutation ? Can we disable mutations ?

Reply:
> If your target platform is similar to the DOM and has methods similar to `appendChild`, `removeChild`, and so on, you'll want to use the **mutation mode**. This is the same mode used by React DOM, React ART, and the classic React Native renderer.
Since I'm using `appendChild` I decided leave it.

Reply:
Isn't `appendChild` called in non mutation mode ?

Reply:
and you are using `appendInitialChild`, not `appendChild`